### ISSUE TITLE
Copy/Language Changes

### ISSUE THREAD

**Issue by Dianadec** [bcgov/cas-registration#552]
#### Describe the task

Update copy/language for all the screens where needed.

#### Acceptance Criteria

- [x] Check and implement the **confirmed language change requests** directly from the [SME Copy Refinement page](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=3422-159&mode=design) in Figma
- [x] Confirm **how each screen should look like for the MVP post changes** from the master [MVP page](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=4-2&mode=design) in Figma
- [x] Nan goes over DEV and check off the change requests in Figma (identify the ones missed/misunderstood, and send to developer to address)

#### Status Update
- Nan has reviewed DEV and recorded the [changes requests missed/misunderstood here](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=5477-6&mode=design&t=VED2zYoBpVjN9ra8-11) for Bri to address
- As discussed, once the above changes are merged into DEV, Bri will inform Nan to go over in DEV again and check them off in Figma

#### Additional context

- Any further copy change requests (that aren't in Figma now) will be implemented post MVP, thus not in the scope of this ticket
- Blocked by #204

**Comment by nanyangpro** [comment]
Hi @BCerki (cc: @hannavovk) - Thanks for your suggestion and volunteering taking this on!

As discussed, we WON'T list all the language change requests in the ticket description. Instead, feel free to
- check the **confirmed language change requests** directly on the [SME Copy Refinement page](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=3422-159&mode=design) in Figma
- confirm **how each screen should look like post changes** on the master [MVP page](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=4-2&mode=design) in Figma

I'm happy to go over the changes with you together in a screensharing session as you suggested. Thanks!

**Comment by BCerki** [comment]
Text changes that are more complicated than changing existing static text are in the following tickets:
- Operations form missing text: https://github.com/orgs/bcgov/projects/122?pane=issue&itemId=50447776
- Breadcrumbs text changes: https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=breadcrumb&pane=issue&itemId=50408649

There's a refactor ticket for the Operator form headings but it doesn't look like any MVP ones are missing: https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=50334427

**Comment by BCerki** [comment]
Note: this part of the app doesn't exist yet:


![Image](https://github.com/bcgov/cas-registration/assets/77306764/3d78a031-aa8a-4b8b-9ef4-18ca384c9865)

So if you want to see it, you can go to the 'Add an Operator' form. We'll conditionally hide it there once the other form is finished.

**Comment by nanyangpro** [comment]
Just an fyi, @hannavovk / @diana - This ticket is NOT ready for PO approval yet.

To make sure that all language/copy change requests from the [SME page](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=3422-159&mode=design) are covered, I'm going over DEV and recording [language/copy divergencies between the implementation](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=5477-6&mode=design) and [latest design](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=4-2&mode=design) today. Once done, I'll inform the developers (cc: @BCerki ) to address them before sending it for PO review and approval.

But feel free to let me know if there are questions! :)

**Comment by nanyangpro** [comment]
Hi @BCerki (cc: @marcellmueller) - I think we were just waiting for this one last piece identified to be merged into DEV, i.e. removing "as Operation Representative" from this page for internal users. If it's in DEV already, feel free to send it to PO review or approval!


![Image](https://github.com/bcgov/cas-registration/assets/58751681/1a44cea6-dc13-4fdc-901d-bfdc7f8cf3b0)
